client/my-sites/people/people-invite-details/test/index.jsx

```diff
@@ -67,10 +76,11 @@ describe( 'PeopleInviteDetails', () => {
 		mockGoBack.mockReset();
 	} );
 
-	it( 'should trigger deletion upon clicking Revoke Invite (pending invite)', () => {
+	it( 'should trigger deletion upon clicking Revoke Invite (pending invite)', async () => {
+		userEvent.setup();
```

Comment:
Does `.setup()` do anything if we don't also use the returned `user` object? Below `userEvent` is used directly which seem to call `.setup()` internally (see the [docs](https://testing-library.com/docs/user-event/setup/#direct-apis) here).